experimental/dds/tree2/src/test/feature-libraries/editable-tree/editableTree.editing.spec.ts

```diff
@@ -383,7 +383,7 @@ describe("editable-tree: editing", () => {
 		trees[0].context.free();
 	});
 
-	describe(`can move nodes`, () => {
+	describe.only(`can move nodes`, () => {
```

Comment:
remove ".only"

Reply:
if you use tree2's vscode workspace and the test browser plugin it recommends it should be easy to run individual tests (from the test browser or clicking the run (or debug) buttons it puts inline above the test. That should remove the need to use .only most of the time

Reply:
removed `.only`